### ISSUE TITLE
my experienment shows that the semantic guided label regularization doesn't work

### ISSUE THREAD

**Issue by akcrazy** [HCPLab-SYSU/HSE#7]
Wish the author to release the training code. My experienments shows the results in the paper may not be achievable. Need the training code to validate its effectiveness
